proto/regen/group/v1alpha1/tx.proto

```diff
@@ -0,0 +1,260 @@
+}
+
+// MsgCreateGroupAccountResponse is the Msg/CreateGroupAccount response type.
+message MsgCreateGroupAccountResponse {
+
+    // group_account is the account address of the newly created group account.
+    bytes group_account = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+}
+
+// MsgUpdateGroupAccountAdminRequest is the Msg/UpdateGroupAccountAdmin request type.
+message MsgUpdateGroupAccountAdminRequest {
+
+    // admin is the account address of the group admin.
+    bytes admin = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    
+    // group_account is the group account address.
+    bytes group_account = 2 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    
+    // new_admin is the new group account admin.
+    bytes new_admin = 3 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+}
+
+// MsgUpdateGroupAccountAdminResponse is the Msg/UpdateGroupAccountAdmin response type.
+message MsgUpdateGroupAccountAdminResponse { }
+
+// MsgUpdateGroupAccountDecisionPolicyRequest is the Msg/UpdateGroupAccountDecisionPolicy request type.
+message MsgUpdateGroupAccountDecisionPolicyRequest {
+    option (gogoproto.goproto_getters) = false;
+
+    // admin is the account address of the group admin.
+    bytes admin = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    
+    // group_account is the group account address.
+    bytes group_account = 2 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    
+    // decision_policy is the updated group account decision policy.
+    google.protobuf.Any decision_policy = 3 [(cosmos_proto.accepts_interface) = "DecisionPolicy"];
+}
+
+// MsgUpdateGroupAccountDecisionPolicyResponse is the Msg/UpdateGroupAccountDecisionPolicy response type.
+message MsgUpdateGroupAccountDecisionPolicyResponse { }
+
+// MsgUpdateGroupAccountCommentRequest is the Msg/UpdateGroupAccountComment request type.
+message MsgUpdateGroupAccountCommentRequest {
+
+    // admin is the account address of the group admin.
+    bytes admin = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    
+    // group_account is the group account address.
+    bytes group_account = 2 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    
+    // comment is the updated group account comment.
+    string comment = 3;
+}
+
+// MsgUpdateGroupAccountCommentResponse is the Msg/UpdateGroupAccountComment response type.
+message MsgUpdateGroupAccountCommentResponse { }
+
+//
+// Proposals and Voting
+//
+
+// MsgCreateProposalRequest is the Msg/CreateProposal request type.
+message MsgCreateProposalRequest {
+    option (gogoproto.goproto_getters) = false;
+
+    // group_account is the group account address.
+    bytes group_account = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
```

Comment:
Similarly, and across all other `types.AccAddress` fields, should this be `string` so as to keep consistent with how we are encoding addresses as bech32 encoded strings elsehwere in the SDK ?